recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "nvcc" %}
-{% set version = environ.get("CUDA_VER", "9.2") %}
+{% set version = environ.get("CUDA_VER", "10.0") %}
```

Comment:
Should we default to 9.2 instead? Things are generally forward compatible but not necessarily backwards compatible.

Reply:
The default here is only to make the linter happy. This is always set to `10.0` by the build.

ref: https://github.com/conda-forge/staged-recipes/pull/8229#pullrequestreview-275703107